projects/components/src/confirmation/confirmation-modal.component.ts

```diff
@@ -52,4 +53,5 @@ export interface ConfirmationModalData {
   confirmButtonLabel?: string;
   confirmButtonRole?: ButtonRole;
   descriptionText: string;
+  highlightParts?: TextHighlightConfig[];
```

Comment:
I don't think this should be part of the confirmation modal. If we want to support completely custom content, we can do that - but I think making confirmation do the customization for us is unsustainable.

Alternative suggestion (we actually discussed doing this when confirmation was being written, but decided to hold off until needed) change `descriptionText` to `description`  or `message` and take `string | TemplateRef`, then use a template outlet to render the template if needed. That leaves the callers full control over the message, including using components, directives, pipes etc.

Reply:
Makes sense. Will drop these changes.